src/components/TextArea/TextArea.tsx

```diff
@@ -0,0 +1,56 @@
+import clsx from 'clsx';
+import React, { forwardRef } from 'react';
+import styles from './TextArea.module.css';
+
+export interface Props
+  extends React.TextareaHTMLAttributes<HTMLTextAreaElement> {
+  /**
+   * CSS class names that can be appended to the component
+   */
+  className?: string;
+  /**
+   * Text default contents of the field
+   */
+  children?: string;
+  /**
+   * Whether the disabled stat is active
+   */
+  disabled?: boolean;
+  /**
+   * Whether the error state is active
+   */
+  isError?: boolean;
+}
+
+/**
+ * Base component, applying styles to a <textarea> tag
+ */
```

Comment:
I've been keeping the beta comment to align with the storybook tag, and how to import the component, which shows up in the Storybook docs page: 
![image](https://user-images.githubusercontent.com/86632227/220749027-1cbc35b0-ab5d-4df6-a039-483cca7f1bba.png)